api/openapi/engine/engine_service.yaml

```diff
@@ -0,0 +1,337 @@
+openapi: 3.0.0
+
+info:
+  title: Engine API
+  version: 0.1.0
```

Comment:
for acquiring knowledge. When do we have to bump this version? when we make any small change in the API right? what happens if, for instance, we add a 500 response line in an endpoint, do we have to bump the version in that case?